### ISSUE TITLE
I will like you to replace this section

### ISSUE THREAD

**Issue by regularshare** [serviceproviderpk/RegularShare#147]
I want products from the buy n selling section to be displaying in the section show in the attached image
so that items posted will have more access on the frontpage
![replace](https://cloud.githubusercontent.com/assets/19783574/18810608/83362c3e-82a3-11e6-9c14-8b943b50ff5d.PNG)

**Comment by serviceproviderpk** [comment]
Home page should be mixup of diffrent things.

on home page of the web site we have already displayed "Products" and product catgory, so we should some other thing in this section.

Please give your suggestion.

Thanks.

**Comment by regularshare** [comment]
Yes I know but there are things that is sell able ,the videos took too much space , the product should have more space in the homepage than the videos

Thanks

**Comment by serviceproviderpk** [comment]
Ok. Fine

**Comment by NasirNiazkiyani** [comment]
Fixed...

**Comment by regularshare** [comment]
I have not seen changes from this topic

**Comment by serviceproviderpk** [comment]
@NasirNiazkiyani  fixed it, I will upload release
